Templates/Main.swifttemplate

```diff
@@ -1,37 +1,28 @@
-<%# ================================================== SETUP -%><%_ -%>
-<%_ var all = types.all
-    all += types.protocols.map { $0 }
-    all += types.protocolCompositions.map { $0 }
+func generate(type: Type, methodRegistrar: MethodRegistrar, subscriptRegistrar: SubscriptRegistrar) async -> String {
```

Comment:
seems to be failing for me:
![Uploading image.png…]()

Reply:
Hey @markst , actually the screenshot looks fine. There's a print statement below, and the contents of the file on the screenshot _is not the final code which would go into the generated mock_

Reply:
it's failing to build the SwiftTemplate:

<img width="463" alt="image" src="https://github.com/user-attachments/assets/efb6dcb5-bd1b-4e14-afef-c54b263dd313">

![Uploading image.png…]()

Reply:
Same for version `2.2.2`:
```
Building for production...
[0/5] Write sources
[2/5] Write swift-version-33747A42983211AE.txt
[4/7] Compiling SourceryRuntime AccessLevel.swift
[5/7] Emitting module SourceryRuntime
error: emit-module command failed with exit code 1 (use -v to see invocation)
[6/9] Emitting module SwiftTemplate
/private/var/folders/qv/k4c8t3_95tzc96m0cpv30tn80000gq/T/SwiftTemplate/B3A5B653-7CFA-413E-A9C8-6074D609F830/2.2.2/Sources/SwiftTemplate/main.swift:12:54: error: cannot find type 'MethodRegistrar' in scope
func generate(type: Type, idx: Int, methodRegistrar: MethodRegistrar, subscriptRegistrar: SubscriptRegistrar) async -> (Int, String) {
                                                     ^~~~~~~~~~~~~~~
/private/var/folders/qv/k4c8t3_95tzc96m0cpv30tn80000gq/T/SwiftTemplate/B3A5B653-7CFA-413E-A9C8-6074D609F830/2.2.2/Sources/SwiftTemplate/main.swift:12:91: error: cannot find type 'SubscriptRegistrar' in scope
func generate(type: Type, idx: Int, methodRegistrar: MethodRegistrar, subscriptRegistrar: SubscriptRegistrar) async -> (Int, String) {
                                                                                          ^~~~~~~~~~~~~~~~~~
/private/var/folders/qv/k4c8t3_95tzc96m0cpv30tn80000gq/T/SwiftTemplate/B3A5B653-7CFA-413E-A9C8-6074D609F830/2.2.2/Sources/SwiftTemplate/main.swift:426:67: error: cannot find 'MethodRegistrar' in scope
            await generate(type: type, idx: idx, methodRegistrar: MethodRegistrar(), subscriptRegistrar: SubscriptRegistrar())
                                                                  ^~~~~~~~~~~~~~~
/private/var/folders/qv/k4c8t3_95tzc96m0cpv30tn80000gq/T/SwiftTemplate/B3A5B653-7CFA-413E-A9C8-6074D609F830/2.2.2/Sources/SwiftTemplate/main.swift:426:106: error: cannot find 'SubscriptRegistrar' in scope
            await generate(type: type, idx: idx, methodRegistrar: MethodRegistrar(), subscriptRegistrar: SubscriptRegistrar())
```